src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private double length;
+    private double width;
+
+    public Rectangle(String color, double width, double length) {
+        super(color);
+        this.width = width;
+        this.length = length;
+    }
+
+    @Override
+    public double getArea() {
+        return length * width;
+    }
+
+    @Override
+    public void draw() {
+        System.out.printf("Figure: rectangle, area: %.2f sq. cm, wight: %.2f cm, length: %.2f cm,"
+                + " color: %s.%n", getArea(), width, length, color);
```

Comment:
The `printf` format string uses 'wight' which seems to be a typo; it should be 'width'. Also, consider using English for all messages in the code to maintain consistency and universality.